docs/quick-ingestion-guides/powerbi/setup.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Setup
+---
+# PowerBI Ingestion Guide: Setup & Prerequisites
+
+In order to configure ingestion from PowerBI, you'll first have to ensure you have an Azure AD app with permission to access the PowerBI resources.
+
+## PowerBI Prerequisites
+
+1. **Create an Azure AD app:** Follow below steps to create an Azure AD app
+
+   a. Login to https://portal.azure.com
+   
+   b. Go to `Azure Active Directory`
+
+   c. Navigate to `App registrations`
+
+   d. Click on `+ New registration`
+
+   e. On `Register an application` window fill the `Name` of application says `powerbi-app-connector` and keep other default as is
+
+      <p align="center">
+   <img width="75%" alt="app_registration" src="https://raw.githubusercontent.com/main/static-assets-fork/main/imgs/guides/powerbi/app-registration.png"/>
+      </p>
+
+   f. On `Register an application` window click on `Register`
+
+   g. The Azure portal will open up the `powerbi-app-connector` window as shown below. On this screen note down the `Application (client) ID` and click on `Add a certificate or secret` to generate a secret for the `Application (client) ID`
+
+      <p align="center">
+   <img width="75%" alt="powerbi_app_connector" src="https://raw.githubusercontent.com/main/static-assets-fork/main/imgs/guides/powerbi/powerbi-connector-window.png"/>
+      </p>
+
+   f. On `powerbi-connector-app | Certificates & secrets` window generate the client secret and note down the `Secret` 
+
+   You need to configure `Application (client) ID` and `Secret` in powerbi connector recipe.
+
+2. **Create an Azure AD Security Group:** You need to add the `Azure AD app` into the security group to control resource permissions for the `Azure AD app`. Follow below steps to create an Azure AD Security Group.
+
+   a. Go to `Azure Active Directory`
+
+   b. Navigate to `Groups` and click on `New group`
+
+   c. On `New group` window fill out the `Group type`,&nbsp; `Group name`, &nbsp;`Group description`. &nbsp;`Group type` should be set to `Security` . &nbsp; `New group` window is shown in below screenshot.
+
+      <p align="center">
+   <img width="75%" alt="powerbi_app_connector" src="https://raw.githubusercontent.com/main/static-assets-fork/main/imgs/guides/powerbi/new-group-window.png"/>
+      </p>
+
+   d. On `New group` window click on `No members selected` and add `Azure AD app` i.e. *powerbi-connector-app* as member
+
+   f. On `New group` window click on `Create` to create the security group `powerbi-connector-app-security-group`.
+
+3. **Assign privileges to powerbi-connector-app-security-group:**  You need to add the created security group into PowerBI portal to allow resource access. Follow below steps to allow `powerbi-connector-app-security-group` to access PowerBI resources.
```

Comment:
let's say "Assign privileges to your security group"

Reply:
done